server/src/main/java/io/littlehorse/server/metrics/PrometheusMetricExporter.java

```diff
@@ -69,19 +62,10 @@ public void bind(Map<String, KafkaStreams> streams) {
         processorMetrics.bindTo(prometheusRegistry);
     }
 
-    public void start() throws IOException {
-        int port = config.getPrometheusExporterPort();
-        String path = config.getPrometheusExporterPath();
-
-        log.info("Starting prometheus service at :{}{}", port, path);
-        server = Javalin.create().get(path, handleRequest()).start(port);
-        log.info("Prometheus started");
-    }
-
-    private Handler handleRequest() {
+    public Handler handleRequest() {
         return ctx -> {
-            log.debug(
-                "Request [from={}, path={}, method={}]",
+            log.trace(
+                "Metrics Request [from={}, path={}, method={}]",
```

Comment:
we agreed on removing this log injection

Reply:
Good catch, I forgot to push the change.